Makefile

```diff
@@ -1,38 +0,0 @@
-#
```

Comment:
Don't remove the original `Makefile`, it's still working to me

Reply:
I'm just removed them because your comment hehe: "... I don't know if I will keep both build systems (Makefile and CMake) or just keep CMake eventually but we're not there yet". But I'm going to restore them in next PR ...